### DISCUSSION TITLE
Any solution to custom resource url

### DISCUSSION THREAD

**Discussion by arunvc** [primefaces/community#2445] [PrimeFaces] [answered]
https://forum.primefaces.org/viewtopic.php?t=646

**Comment by melloware** [comment, marked as answer]
Yes you can easily plug in your own ResourceHandler in Faces.

See:
https://github.com/primefaces/primefaces/blob/master/primefaces/src/main/java/org/primefaces/application/resource/PrimeResourceHandler.java

https://github.com/primefaces/primefaces/blob/master/primefaces/src/main/java/org/primefaces/application/resource/PrimeResource.java

Specifically in PrimeResource you probably want to override the method...

```java
    @Override
    public String getRequestPath() {
        if (version == null) {
            return super.getRequestPath();
        }
        return super.getRequestPath() + version;
    }
```
